assignments/objects.js

```diff
@@ -49,16 +99,34 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
```

Comment:
Great job! I would suggest some tabbing to make it a little easier to read. ``` const parent = {
  "name": "Susan", 
  "age": 70, 
    "child": {
      "name": "George", 
      "age": 50
    }, 
        "grandchild": {
          "name": "Sam", 
        "age": 30
      }, ``` 

Just so someone can glance at and see that it's nested. Other than, great job!